corehq/form_processor/models.py

```diff
@@ -563,6 +563,11 @@ def is_deleted(self):
     def dynamic_case_properties(self):
         return OrderedDict(sorted(self.case_json.iteritems()))
 
+    def get_json(self, lite=False):
```

Comment:
https://github.com/dimagi/commcare-hq/pull/11231/commits/171c1950eb43326e0399a9c78550c65048b9fe2f - wasn't actually that hard